powerfactory/powerfactory-converter/src/main/java/com/powsybl/powerfactory/converter/PowerFactoryImporter.java

```diff
@@ -159,11 +159,33 @@ private Network createNetwork(StudyCase studyCase, NetworkFactory networkFactory
             }
         }
 
-        LOGGER.info("Creating equipments...");
+        LOGGER.info("Creating equipment...");
 
+        List<DataObject> slackObjects = new ArrayList<>();
+
+        // Create main equipment
+        creatingMaintEquipment(studyCase, importContext, hvdcConverter, network, slackObjects);
```

Comment:
consider renaming the method to `convertEquipment` (`create` instead of creating; remove `main` because it does not add meaning; fix typo in `...MaintE...`, there is a letter `t` not needed)

Reply:
Done